foundation-extensions/experience-platform-extension/release-notes.md

```diff
@@ -1,5 +1,15 @@
 # Release Notes
 
+## Sept 2, 2021
+
+### iOS AEPEdge 1.1.2
+
+* Now honors timestamp set in XDM payload of Experience Event. If no timestamp is set then timestamp of `Edge.sendEvent()` API call is used.
```

Comment:
Edge Network extension now honors the timestamp set in XDM payload of Experience event. If no timestamp is set, then timestamp of `Edge.sendEvent()` API call is used.